### PR TITLE
MODSOURMAN-453:  Add index for the " job_execution_source_chunks"

### PR DESCRIPTION
**Pull request by VRohach**
## Purpose
Increase performance and remove extra logging

## Approach
Full-text search changed on equals search.
Add 2 indices.

## Learning
More info: https://issues.folio.org/browse/MODSOURMAN-453

### CODE REVIEW

**Review by RuslanLavrov — APPROVED** [review]

File: mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "tables": [
     {
       "tableName": "job_executions",
-      "fromModuleVersion": "mod-source-record-manager-3.0.5",
+      "fromModuleVersion": "mod-source-record-manager-3.0.6",
```
Comment (KaterynaSenchenko) [inline mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json:5]: By default it won't be released as a hotfix, please use the 3.1.0 version. If it is crucial for Iris, then we'll change it

File: mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json [hunk 2]
```diff
@@ -26,7 +29,7 @@
     },
     {
       "tableName": "job_execution_source_chunks",
-      "fromModuleVersion": "mod-source-record-manager-1.2.0",
+      "fromModuleVersion": "mod-source-record-manager-3.0.6",
```
Comment (KaterynaSenchenko) [inline mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json:32]: Same here

File: mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json (hunk 1 above)
Comment (VRohach) [reply mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json]: Changed

File: mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json (hunk 2 above)
Comment (VRohach) [reply mod-source-record-manager-server/src/main/resources/templates/db_scripts/schema.json]: Changed

**Review by KaterynaSenchenko — APPROVED** [review]

**Pull request merged**